nupic/frameworks/opf/clamodel.py

```diff
@@ -664,13 +663,10 @@ def _handleCLAClassifierMultiStep(self, patternNZ,
                   None.
     rawInput:   The raw input to the sensor, as a dict.
     """
-    inferenceArgs = self.getInferenceArgs()
-    predictedFieldName = inferenceArgs.get('predictedField', None)
-    if predictedFieldName is None:
+    if self._predictedFieldName is None:
```

Comment:
You are removing the code that sets `self._predictedFieldName` so where does it get set now?